CHANGES.rst

```diff
@@ -10,6 +10,188 @@
 
 .. towncrier release notes start
 
+3.9.2 (2024-01-28)
+==================
+
+Bug fixes
+---------
+
+- Fixed websocket connection leak.
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`7978`.
+
+
+
```

Comment:
@webknjaz If you could check the template code for this. These 3 lines include whitespace "  " in each section. I manually deleted them before commit.

Reply:
I think that it's one of the artifacts of Jinja templating. Things like `{%- endwith -%}` drop spaces before and after the instruction, while the tag variants without those `-` keep them. It's rather tricky to get it right for all of the combinations of reference paragraphs.
I don't remember exactly, but I must've ended up with this compromise.

It's hard to debug and it was far more important to me that the rendered web page would look good, even at the cost of RST being less pretty.

Maybe, at some point I'll get to updating this, but right now it's not a priority for me.

If you're curious, you can take a look yourself, but that requires *manual* testing of variants with just commit hashes, just PR/issue numbers, just arbitrary references that are neither commit hashes nor numbers, and combinations of all or part of those together.

But I don't recommend doing this right now because it can be quite a lengthy trip down the rabbit hole.

Reply:
Right, at a glance, I'd think that the change would need to be from:
`{% if something -%}` to `{%- if something %}
At:
https://github.com/aio-libs/aiohttp/blob/master/CHANGES/.TEMPLATE.rst?plain=1#L48-L61

i.e. Keep the newline that's inside the block, but lose the newline with "  " before the block.

Reply:
Yes, but I remember it breaking in some corner cases which irritated me enough to postpone figuring out how to make it work better.

Reply:
I'd very much like to see this improved, but it's annoying to debug so I temporarily gave up. There's some important work to do in multidict that blocks it from being released, which I think takes priority.